src/lib/storage/value_segment.cpp

```diff
@@ -17,18 +17,22 @@ template <typename T>
 AllTypeVariant ValueSegment<T>::operator[](const ChunkOffset chunk_offset) const {
   PerformanceWarning("operator[] used");
 
-  throw std::runtime_error("Implement ValueSegment::operator[]");
+  return _data[chunk_offset];
```

Comment:
maybe add
`DebugAssert(chunk_offset < m_values.size(), "Assert in ValueSegment::operator[]: chunk_offset doesn't fit into vector of values.");`